tests/app/questionnaire/test_router.py

```diff
@@ -1566,6 +1567,7 @@ def test_return_to_grand_calculated_summary_from_repeating_answer(
             return_to="grand-calculated-summary",
             block_id="calculated-summary-6",
             return_to_block_id="grand-calculated-summary-3",
+            return_to_answer_id="calculated-summary-6,calculated-summary-6",
```

Comment:
👀

Reply:
Removed return answer id now as it shouldn't have one in url?